packages/webpack-plugin/src/plugin.ts

```diff
@@ -295,40 +295,38 @@ export class StylableWebpackPlugin {
         const options =
             loadStylableConfig(compiler.context, (config) => {
                 return isWebpackConfigProcessor(config)
-                    ? config.webpackPlugin(defaults, compiler)
+                    ? config.webpackPlugin(defaults, compiler, getTopLevelInputFilesystem(compiler))
                     : undefined;
             })?.config || defaults;
 
         this.options = options;
     }
-    private createStcBuilder(compiler: Compiler) {
-        if (!this.options.stcConfig) {
-            return;
-        }
-
+    private getStylableConfig(compiler: Compiler) {
         const configuration = resolveStcConfig(
             compiler.context,
-            typeof this.options.stcConfig === 'string' ? this.options.stcConfig : undefined
+            typeof this.options.stcConfig === 'string' ? this.options.stcConfig : undefined,
+            getTopLevelInputFilesystem(compiler)
         );
 
-        if (!configuration) {
```

Comment:
where did this error go?

Reply:
`resolveStcConfig` actually has a tryRun internally, so the check here was redundant.